Gemfile

```
@@ -60,7 +60,9 @@ end
 
 group :test do
   platforms(*(@windows_platforms + [:ruby])) do
-    if version == 'master' || version >= '6'
+    if version >= '8'
+      gem 'sqlite3', '~> 2.1'
+    elsif version == 'master' || version >= '6'
```

would it be this way?

```suggestion
    if version == 'master' || version >= '8'
      gem 'sqlite3', '~> 2.1'
    elsif version >= '6'
```